modules/ROOT/nav.adoc

```diff
@@ -74,6 +74,7 @@
 **** xref:deploy:deployment-option/cloud/manage-billing/aws-commit.adoc[Use AWS Commits]
 *** xref:deploy:deployment-option/cloud/security/index.adoc[Security]
 **** xref:deploy:deployment-option/cloud/security/cloud-authentication.adoc[Authentication]
+**** xref:deploy:deployment-option/cloud/security/cloud-enable-mtls-kafka-api.adoc[Enable mTLS for Kafka API]
```

Comment:
@paulzhang97 @micheleRP would like to double check if this seems like an appropriate place in the navigation tree for this doc.

Reply:
@paulzhang97 This is an authentication option, and should appear as a subtopic under AuthN, no? Makes no sense to me that it is at the same level as AuthN.

Reply:
@Feediver1 -- `Security/Authentication` in the nav tree is currently a conceptual doc https://docs.redpanda.com/current/deploy/deployment-option/cloud/security/cloud-authentication/ whereas this new content is a task-oriented topic, so I've added it as its own doc. I don't know if we would actually want to turn `Security/Authentication` as its own subsection, perhaps I'll defer to @micheleRP ?

Reply:
I don't have a strong opinion about where it lands right now, since we'll be changing so much soon with the Cloud docs reorg. @paulzhang97 or @Feediver1: would you prefer that this content be incorporated into the Authentication page as  a nested section in the [Service authentication section](https://deploy-preview-520--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/security/cloud-authentication/#service-authentication) or kept as a standalone file but nested under [Authentication](https://deploy-preview-520--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/security/)?

Reply:
@paulzhang97  I found it odd that this is placed outside the scope of the authN heading/umbrella here--thoughts?

Reply:
I would like to put it under [Service authentication section](https://deploy-preview-520--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/security/cloud-authentication/#service-authentication). It sounds better fit to me.

Reply:
Merge into the requested page.